src/runtime/ProxyObject.cpp

```diff
@@ -338,8 +338,16 @@ ObjectGetResult ProxyObject::getOwnProperty(ExecutionState& state, const ObjectP
             // i. Throw a TypeError exception.
             ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, "Proxy::getOwnPropertyDescriptor error");
         }
+        // b. If resultDesc has a [[Writable]] field and resultDesc.[[Writable]] is false, then
+        if (!resultDesc.isWritable()) {
```

Comment:
IMO It would be more accurate to fix as follow.
```suggestion
        if (resultDesc.isWritablePresent() && !resultDesc.isWritable()) {
```
BTW it seems that this new feature is part of ECMA draft not es10, right?

Reply:
That's true, if you want i can revert the changes.

Reply:
Yes, please do so. We will start adding the new features when the new standard released. For now, we focus on ECMAScript 2019. Other part of this pr looks fine to me:)